doc/sphinx/Pacemaker_Explained/operations.rst

```diff
@@ -38,113 +38,160 @@ two operations for the same resource with the same name and interval.
-   |                |                                   | makes sense for recurring ``monitor`` operations.   |
-   |                |                                   | Allowed (case-sensitive) values: ``Stopped``,       |
-   |                |                                   | ``Started``, and in the case of :ref:`promotable    |
-   |                |                                   | clone resources <s-resource-promotable>`,           |
-   |                |                                   | ``Unpromoted`` and ``Promoted``.                    |
-   +----------------+-----------------------------------+-----------------------------------------------------+
+   :widths: 2 2 3 4
+   :header-rows: 1
+
+   * - Name
+     - Type
+     - Default
+     - Description
+   * - .. _op_id:
+       
+       .. index::
+          pair: op; id
+          single: id; action property
+          single: action; property, id
+       
+       id
+     - :ref:`id <id>`
+     - 
+     - A unique identifier for the XML element *(required)*
+   * - .. _op_name:
+       
+       .. index::
+          pair: op; name
+          single: name; action property
+          single: action; property, name
+       
+       name
+     - :ref:`text <text>`
+     - 
+     - An action name supported by the resource agent *(required)*
+   * - .. _op_interval:
+       
+       .. index::
+          pair: op; interval
+          single: interval; action property
+          single: action; property, interval
+       
+       interval
+     - :ref:`duration <duration>`
+     - 0
+     - A positive value directs Pacemaker to schedule instances of the action
+       that recur at the given interval, which makes sense only with ``name``
+       set to :ref:`monitor <s-resource-monitoring>`. A value of 0 directs
+       Pacemaker to apply any other properties configured for this action to
+       nonrecurring instances scheduled as needed. *(required)*
+   * - .. _op_role:
+       
+       .. index::
+          pair: op; role
+          single: role; action property
+          single: action; property, role
+       
+       role
+     - :ref:`enumeration <enumeration>`
+     - 
+     - If this is set, the operation configuration applies only on nodes where
+       the cluster expects the resource to be in the specified role. This makes
+       sense only for recurring monitors. Allowed values: ``Started``,
+       ``Stopped``, and in the case of :ref:`promotable clone resources
+       <s-resource-promotable>`, ``Unpromoted`` and ``Promoted``.
+   * - .. _op_timeout:
+       
+       .. index::
+          pair: op; timeout
+          single: timeout; action property
+          single: action; property, timeout
+       
+       timeout
+     - :ref:`timeout <timeout>`
+     - 20s
+     - If resource agent execution does not complete within this amount of
+       time, the action will be considered failed. **Note:** timeouts for
+       fencing agents are configured differently (see
```

Comment:
Yes and no. For fence agents, the special timeouts really apply only to starts and monitors. Also, if the special instance attribute is not set, then the usual method of configuring timeouts (via `<op>` attributes, etc.) still applies. See `pcmk__unpack_action_meta()`.

Reply:
I'm just going to point to the whole fencing chapter here for now, and leave improving the timeout section there for later